TDDPOC/TDDPOC.Test/WeatherForecastTest.cs

```diff
@@ -0,0 +1,64 @@
+using AutoFixture;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using TDDPOC.Controllers;
+using TDDPOC.Service;
+
+namespace TDDPOC.Test
+{
+    public class WeatherForecastTest
+    {
+        private WeatherForecastController _controller;
+        private IFixture _fixture;
+        private Mock<IWeatherForecastService> _mockWeatherForecastService;
+
+        public WeatherForecastTest()
+        {
+            _fixture = new Fixture();
+            _mockWeatherForecastService = new Mock<IWeatherForecastService>();
+            _fixture.Register(() => _mockWeatherForecastService.Object);
+            _controller = new WeatherForecastController(_mockWeatherForecastService.Object);
+        }
+
+        [Fact]
+        public async void Get_WeatherForeCasts_ReturnsWeather()
+        {
+            var weatherForecast = _fixture.Create<WeatherForecast>();
+            _mockWeatherForecastService.Setup(w => w.GetById(weatherForecast.Summary)).Returns(weatherForecast);
+
+            var result = await _controller.GetBySummary(weatherForecast.Summary);
+
+            var objectResult = result.Should().BeOfType<OkObjectResult>().Subject;
+            objectResult.StatusCode.Should().Be(200);
+            var actualValue = objectResult.Value.Should().BeOfType<WeatherForecast>().Subject;
+            actualValue.Should().NotBeNull();
```

Comment:
Not neeeded as we are directly verifying the actual response

Reply:
removed